forch/faucet_state_collector.py

```diff
@@ -1380,7 +1380,7 @@ def _get_port_attributes(self, switch, port):
                 return ret_attr
 
             sequester_port_desc = self._config.orchestration.sequester_config.port_description
-            if sequester_port_desc and sequester_port_desc in port_info.get('description', ''):
+            if sequester_port_desc and sequester_port_desc == port_info.get('description', ''):
```

Comment:
don't need the .get() with default (since it would default to None anyway)

Reply:
Done